tools/api-client/python/create_game

```diff
@@ -90,15 +90,15 @@ def create_games(b, buttons, opponents):
   for opponent in opponents:
     pbutton = random_array_element(buttons)
     obutton = random_array_element(buttons)
-    print "%s (%s) vs. %s (%s)" % (b.username, pbutton, opponent, obutton)
+    print ("%s (%s) vs. %s (%s)" % (b.username, pbutton, opponent, obutton))
     time.sleep(3)
-    print b.wrap_create_game(pbutton, obutton, opponent)
+    print (b.wrap_create_game(pbutton, obutton, opponent))
 
 def create_random_game(opponent, b, players=False, usebuttons=False):
   if not players:
     players = lookup_other_players(b)
   if not opponent in players:
-    raise ValueError, "Opponent %s is not known on site" % opponent
+    raise (ValueError, "Opponent %s is not known on site" % opponent)
```

Comment:
Hmm, do we like `raise (ValueError` or `raise(ValueError` - i've usually done the latter (and would presumably do the same with print()), on the theory that if it looks like a function, it should be whitespaced like a function.  Whatcha think?